web/client/components/data/identify/IdentifyContainer.jsx

```diff
@@ -76,10 +78,10 @@ module.exports = props => {
         lngCorrected = lngCorrected - 360 * Math.floor(lngCorrected / 360 + 0.5);
     }
     const Viewer = viewer;
-    const buttons = getButtons({...props, lngCorrected, validResponses, latlng});
+    // TODO: put all the header (Toolbar, navigation, coordinate editor) outside the container
+    const buttons = getToolButtons({...props, lngCorrected, validResponses, latlng});
```

Comment:
let's call it toolsButtons or buttonsTools

Reply:
:ok: